### PR TITLE
Southeast Militia ---> Free Worlds

### PR DESCRIPTION
**Pull request by toilethinges**
Changing plain Militia to FW Militia in pirate systems.

I probably missed several places, and the initial deployment stuff seemed a bit weird, so I made it as close as I could reason. Please tell me where I did miss stuff.

Root: #1496 

This has turned from what I thought was a bug fix to what I would tag "Enhancement." Now it's just changing from Militia fleets everywhere to Free Worlds, because of Bloodsea's modified description:

> description `Until recently, this was a pirate world, but they are now officially "independent," forswearing support for piracy and other criminal activity as a result of being dominated by the Free Worlds militia. You get the feeling that most of the locals are not too fond of the Free Worlds, however.`

All the Militia is under the command of the Free Worlds (I thought/think), so it would make sense that the Militia would be Free Worlds government rather than just independent Militia government.

### CODE REVIEW

**Comment by toilethinges** [comment]
Hopefully, that's all.

File: data/events.txt [hunk 1]
```diff
@@ -1382,6 +1382,14 @@ event "initial deployment 4"
...
 		fleet "Small Free Worlds" 3000
 		fleet "Large Free Worlds" 5000
+	system "Shaula"
+		fleet "Small Southern Pirates" 500
+		fleet "Large Southern Pirates" 800
+		fleet "Small Core Pirates" 4000
+		fleet "Large Core Pirates" 7000
+		fleet "Small Northern Pirates" 5000
+		fleet "Large Northern Pirates" 9000
+		fleet "Large Militia" 9000
```
Comment (endless-sky) [inline data/events.txt]: This is identical to the default fleets for Shaula, and this is the first event that would change those fleets, so I don't think it's necessary.

Comment (toilethinges) [inline data/events.txt]: I think that was meant to be Free Worlds, but I haven't looked at this in a long time, and I can't remember.

File: data/events.txt [hunk 2]
```diff
@@ -487,13 +487,70 @@ event "initial deployment 3"
...
+		fleet "Small Southern Merchants" 600
+		fleet "Large Southern Merchants" 1000
+		fleet "Small Free Worlds" 4000
+		fleet "Small Southern Pirates" 3000
+	system "Naper"
+		fleet "Small Southern Merchants" 1500
+		fleet "Large Southern Merchants" 3000
+		fleet "Small Free Worlds" 5000
+		fleet "Small Southern Pirates" 1000
+	system ""
```
Comment (Amazinite) [inline data/events.txt]: ?

**Comment by toilethinges** [comment]
Closing because I don't expect it to be merged, and don't want to update it any more.

**Pull request closed without merging**